### DISCUSSION TITLE
overleaf模板

### DISCUSSION THREAD

**Discussion by yupei9** [ustctug/ustcthesis#297] [Q&A]
overleaf模板编译出来的中文字体是等线，怎么设置中文宋体

**Comment by sikouhjw** [comment]
https://github.com/whutug/whu-thesis/issues/56#issuecomment-821925096

**Comment by sikouhjw** [comment]
这哪里等线了，一看就思源啊……
https://github.com/ustctug/ustcthesis/blob/b772e306cb644e87187e24c4ed8378ee6b9b3827/ustcthesis.cls#L655-L669
https://github.com/ustctug/ustcthesis/blob/b772e306cb644e87187e24c4ed8378ee6b9b3827/ustcthesis.cls#L904-L918

**Reply by tanukihee** [reply]
等线是商业字体，overleaf 上怎么可能会有（

**Reply by sikouhjw** [reply]
> 
> 
> 等线是商业字体，overleaf 上怎么可能会有（

是思源，他加了思源（

**Comment by zepinglee** [comment]
<img width="337" alt="Screen Shot 2021-04-21 at 23 50 08" src="https://user-images.githubusercontent.com/12290822/115583519-8a426f00-a2fc-11eb-8adb-083baecfac9d.png">
这俩差得老远了，一个有衬线一个没衬线。
